_includes/guide/patient_resource.html

```diff
@@ -1,3 +1,3 @@
 <p>
-	The <a href="https://www.hl7.org/fhir/patient.html" target="_blank" class="in-text__link"> Patient</a> resource type can be thought of as an ACO’s CCLF files 8 and 9: this is where you get information about who your beneficiaries are, their demographic information, and updates to their patient identifiers.
+	The <a href="https://www.hl7.org/fhir/patient.html" target="_blank" class="in-text__link"> Patient</a> The Patient resource type can be thought of as your CCLF files 8 and 9. This is where you get your information about who your beneficiaries are, their demographic information, and updates to their patient identifiers.
```

Comment:
Duplicate wording, this should be `The <a href="https://www.hl7.org/fhir/patient.html" target="_blank" class="in-text__link"> Patient</a> resource type can be thought of as your CCLF files 8 and 9. This is where you get your information about who your beneficiaries are, their demographic information, and updates to their patient identifiers.`